data/resources.yml

```
@@ -0,0 +1,31 @@
+resources:
+  - title:  Learn-OCaml
+    short_description: This is Learn-OCaml, a platform for learning the OCaml language, featuring a Web toplevel, an exercise environment.
+    long_description: A platform designed for learning the OCaml programming language. It features a web-based interactive toplevel, an environment for exercises, and a collection of lessons and tutorials. The platform is suitable for both beginners and experienced programmers and can be used for academic purposes or self-study.
```

Just a small suggestion, to highlight one of the most typical learn-ocaml feature:

```suggestion
    long_description: A platform designed for learning the OCaml programming language. It features a web-based interactive toplevel, an environment for exercises with automated grading tests, and a collection of lessons and tutorials. The platform is suitable for both beginners and experienced programmers and can be used for academic purposes or self-study.
```